dbt_schema_builder/builder.py

```diff
@@ -616,4 +621,7 @@ def run(self):
             for app_name, app_config in self.builder.app_schema_configs.items():
                 logger.info('\n')
                 logger.info('------- {} -------'.format(app_name))
-                self.builder.build_app(app_name, app_config)
+                if no_pii == True:
```

Comment:
You don't even need the `if` statement here, you can just pass `self.builder.build_app(app_name, app_config, no_pii=no_pii)` and it will pass through

Reply:
@dnuzz Ty, implemented here and elsewhere in my code